c10/xpu/XPUException.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <c10/util/Exception.h>
+#include <sycl/sycl.hpp>
+
+namespace c10::xpu {
+
+static sycl::async_handler asyncHandler = [](sycl::exception_list eL) {
```

Comment:
e**L** --> el ?

Reply:
following other's code style, see https://github.com/codeplaysoftware/computecpp-sdk/blob/fb9b7bab63511cab1a41219618d94db4246eddda/samples/async-handler.cpp#L70

Reply:
we should follow pytorch style. pls check in pytorch if has same coding?

Reply:
done.